src/GitTools.Testing/Fixtures/SequenceDiagram.cs

```diff
@@ -9,89 +9,74 @@ namespace GitTools.Testing
     /// </summary>
     public class SequenceDiagram
     {
-        readonly Dictionary<string, string> _participants = new Dictionary<string, string>();
-        readonly StringBuilder _diagramBuilder;
+        private readonly Dictionary<string, string> participants = new();
```

Comment:
Forgot to mention I went ahead to use the new syntax and to favor it but I can change that.